apps/ui/src/networks/offchain/api/index.ts

```diff
@@ -412,13 +413,63 @@ export function createApi(uri: string, networkId: NetworkID): NetworkApi {
         farcaster: user.farcaster || ''
       };
     },
-    loadUserActivities: async (): Promise<UserActivity[]> => {
-      // NOTE: leaderboard implementation is pending on offchain
-      return [];
+    loadUserActivities(userId: string): Promise<UserActivity[]> {
+      return apollo
+        .query({
+          query: LEADERBOARD_QUERY,
+          variables: {
+            first: 1000,
+            skip: 0,
+            orderBy: 'proposal_count',
+            orderDirection: 'desc',
+            where: {
+              user: userId
+            }
+          }
+        })
+        .then(({ data }) =>
+          data.leaderboards.map((leaderboard: any) => ({
+            spaceId: `${networkId}:${leaderboard.space.id}`,
+            vote_count: leaderboard.vote_count,
+            proposal_count: leaderboard.proposal_count
+          }))
+        );
```

Comment:
This looks wrong, was this intended? It's for user activity, not leaderboard.

Reply:
Leaderboard query is below.

This PR unblock everything related to leaderboard on offchain, so will also enable user activities on user profile page

Reply:
Those both are the same queries `LEADERBOARD_QUERY`. And user activities do not work.